### DISCUSSION TITLE
Example in documentation uses the wrong version parameter?

### DISCUSSION THREAD

**Discussion by ralphhaselbach** [svrooij/WinTuner#51] [General]
The section [**Update all and supersede the old ones**](https://wintuner.app/docs/wintuner-powershell/Get-WtWin32Apps#update-all-and-supersede-the-old-ones) in the documentation of Get-WtWin32App (on wintuner.app) states that the parameter `-Version $($app.CurrentVersion)` should be used for the New-WtWingetPackage command.

Shouldn't that be `-Version $($app.LatestVersion)`?

**Comment by svrooij** [comment]
You're absolutely right, will be fixed in a couple of minutes ✅
